doc/mmp_testcases.txt

```diff
@@ -0,0 +1,66 @@
+Poor Man's Multiple Mount Protection Test Cases
+
+Variables to Consider
+* Host ID
+* Activity Present
+* VDEV state
+* zpool import -f
+* Tunable zfs_mmp_interval
+
+Notes abut zfs_mmp_interval
+* zfs_mmp_interval is measured in milliseconds. The write frequency is
+  number of leaf vdevs / zfs_mmp_interval.
+* Setting zfs_mmp_interval to 0, will disable mmp. No mmp uberblock updates
+  should occur and no activity checking on zpool import.
+
+Test Cases
+* (done) Create a pool with single VDEV, monitor VDEV's labels for changes
+  to the uberblocks.
+* Create a pool with single VDEV, monitor VDEV's labels for changes
+  to the uberblocks. Destroy the pool, verify that the MMP update thread stops.
+* Create a pool with single VDEV, monitor VDEV's labels for changes
+  to the uberblocks. Export the pool, verify that the MMP update thread stops.
+* Import a pool with single VDEV, monitor VDEV's labels for changes
+  to the uberblocks. Destroy the pool, verify that the MMP update thread stops.
```

Comment:
At least 3 times now, I've introduced bugs that caused the mmp_thread_stop() to fail or hang, so that the mmp thread did not go away.

I think we should either have one test that verifies the mmp thread started and stopped, or just add those checks to the existing import/export tests.

The real contract with the user, it seems to me, is that the mmp writes happen when they are supposed to.  However, if the mmp thread handling is buggy, I expect it would impact the user in ways that they do not understand (e.g. export hangs).  So I think the test is worthwhile anyway.